Src/CSharpier.Cli/CommandLineFormatter.cs

```diff
@@ -155,7 +155,7 @@ CancellationToken cancellationToken
                 : directoryOrFilePath;
 
             var optionsProvider = await OptionsProvider.Create(
-                directoryName,
+                directoryName!,
```

Comment:
I really don't like the null forgiving `!` operator.
Would it be feasible to put null checks in all these instead?

Reply:
Yeah i can do that.